libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteLayerProvider.java

```diff
@@ -29,72 +28,85 @@
 import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineColor;
 import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineJoin;
 import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineWidth;
+import static com.mapbox.navigation.ui.route.RouteConstants.ALTERNATIVE_ROUTE_LAYER_ID;
+import static com.mapbox.navigation.ui.route.RouteConstants.ALTERNATIVE_ROUTE_SHIELD_LAYER_ID;
+import static com.mapbox.navigation.ui.route.RouteConstants.ALTERNATIVE_ROUTE_SOURCE_ID;
 import static com.mapbox.navigation.ui.route.RouteConstants.DESTINATION_MARKER_NAME;
-import static com.mapbox.navigation.ui.route.RouteConstants.HEAVY_CONGESTION_VALUE;
-import static com.mapbox.navigation.ui.route.RouteConstants.MODERATE_CONGESTION_VALUE;
 import static com.mapbox.navigation.ui.route.RouteConstants.ORIGIN_MARKER_NAME;
-import static com.mapbox.navigation.ui.route.RouteConstants.PRIMARY_ROUTE_PROPERTY_KEY;
-import static com.mapbox.navigation.ui.route.RouteConstants.ROUTE_LAYER_ID;
-import static com.mapbox.navigation.ui.route.RouteConstants.ROUTE_SHIELD_LAYER_ID;
-import static com.mapbox.navigation.ui.route.RouteConstants.ROUTE_SOURCE_ID;
-import static com.mapbox.navigation.ui.route.RouteConstants.SEVERE_CONGESTION_VALUE;
+import static com.mapbox.navigation.ui.route.RouteConstants.PRIMARY_ROUTE_LAYER_ID;
+import static com.mapbox.navigation.ui.route.RouteConstants.PRIMARY_ROUTE_SHIELD_LAYER_ID;
+import static com.mapbox.navigation.ui.route.RouteConstants.PRIMARY_ROUTE_SOURCE_ID;
 import static com.mapbox.navigation.ui.route.RouteConstants.WAYPOINT_DESTINATION_VALUE;
 import static com.mapbox.navigation.ui.route.RouteConstants.WAYPOINT_LAYER_ID;
 import static com.mapbox.navigation.ui.route.RouteConstants.WAYPOINT_ORIGIN_VALUE;
 import static com.mapbox.navigation.ui.route.RouteConstants.WAYPOINT_PROPERTY_KEY;
 import static com.mapbox.navigation.ui.route.RouteConstants.WAYPOINT_SOURCE_ID;
 
-class MapRouteLayerProvider {
+public class MapRouteLayerProvider {
```

Comment:
Does this class need to be public?

Reply:
I can't access it from the MapRouteLine class if it's not public. The compiler complains "public function exposes its public/*package*/ parameter type MapRouteLayerProvider

Reply:
This is a good candidate to move under an `internal` package 😉 cc @abhishek1508